RecoTauTag/HLTProducers/src/L1PFJetsMatching.cc

```diff
@@ -0,0 +1,133 @@
+#include "RecoTauTag/HLTProducers/interface/L1PFJetsMatching.h"
+#include "Math/GenVector/VectorUtil.h"
+#include "DataFormats/HLTReco/interface/TriggerTypeDefs.h"
+#include "FWCore/Utilities/interface/EDMException.h"
+#include "DataFormats/JetReco/interface/PFJet.h"
+
+//
+// class decleration
+//
+using namespace reco   ;
+using namespace std    ;
+using namespace edm    ;
+using namespace trigger;
+
+std::pair<PFJetCollection,PFJetCollection> categorise(PFJetCollection PFMatchedJets, double pt1, double pt2, double Mjj)
+{
+    std::pair<PFJetCollection,PFJetCollection> Output;
+    unsigned int i1 = 0;
+    unsigned int i2 = 0;
+    double mjj = 0;
+    if (PFMatchedJets.size()>1){
+    for (unsigned int i = 0; i < PFMatchedJets.size()-1; i++)
+        for (unsigned int j = i+1; j < PFMatchedJets.size(); j++)
+    {
+        const PFJet &  myJet1 = (PFMatchedJets)[i];
+        const PFJet &  myJet2 = (PFMatchedJets)[j];
+        
+        
+        if ((myJet1.p4()+myJet2.p4()).M()>mjj){
+            
+            mjj =(myJet1.p4()+myJet2.p4()).M();
+            i1 = i;
+            i2 = j;
+        }
+    }
+        
+            const PFJet &  myJet1 = (PFMatchedJets)[i1];
+            const PFJet &  myJet2 = (PFMatchedJets)[i2];
+        
+        if ((myJet1.p4().Pt() >= pt1) && (myJet2.p4().Pt() > pt2) && (mjj > Mjj))
+        {
+            
+            Output.first.push_back(myJet1);
+            Output.first.push_back(myJet2);
+            
+	}
+        
+        if ((myJet1.p4().Pt() < pt1) && (myJet1.p4().Pt() > pt2) && (myJet2.p4().Pt() > pt2) && (mjj > Mjj))
+        {
+            
+            const PFJet &  myJetTest = (PFMatchedJets)[0];
+         if (myJetTest.p4().Pt()>pt1){
+            Output.second.push_back(myJet1);
+            Output.second.push_back(myJet2);
+            Output.second.push_back(myJetTest);
+             
+		}
+        }
+        
+    }
+    
+            return Output;
+        
+}
+
```

Comment:
Please fix the indentation of the ```categorise``` function code block - currently that code is hard to read (lines 15 to 65).